src/Workspaces/Remote/Razor/Shared/WatsonReporter.cs

```diff
@@ -1,47 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
What's going on with this file?

Reply:
I think we added it because we didn't flow some of the references correctly and were missing the types, or something. Turns out we are not using the types anymore anyways and the definitions just caused ambiguities. So I deleted it.